kubernetes-services/work-scheduler/test/helpers/env.ts

```diff
@@ -0,0 +1,3 @@
+process.env.SHARED_SECRET_KEY = '_THIS_IS_MY_32_CHARS_SECRET_KEY_';
+process.env.NODE_ENV = 'test';
+process.env.ARTIFACT_BUCKET = 'stac-catalogs';
```

Comment:
Why are we using the artifact bucket?

Reply:
Because I copied this from the `service-runner` directory. Removed.